theme/404.html

```diff
@@ -7,9 +7,10 @@ <h1>Page not found.</h1>
     <h2>You probably weren't expecting to land here.</h2>
     <h3>Here are some useful links instead:</h3>
     <ul>
-        <li><a href="/creating_content_basic/">Creating content</a></li>
-        <li><a href="/publishing/publishing/">Publishing</a></li>
-        <li><a href="/search/">Searching</a></li>
+        <li><a href="/projects/userguide/en/latest/content_management/create_content_basic/">Creating content</a></li>
+        <li><a href="/projects/userguide/en/latest/content_management/content_publishing/publishing/">Publishing</a></li>
```

Comment:
These two links still lead to another 404

Reply:
Thank you! Will fix as we speak